admin-dev/themes/default/template/controllers/access/helpers/form/form.tpl

```diff
@@ -67,6 +78,16 @@
          var classes = $parentRow.attr('class');
          var $permissionCheckbox = $(this);
          var targetPermissionType;
+         var parent = $(this).attr('data-parent');
+         if (parent != 0){
+            var $parentelem = $(table + ' .ajaxPower.' + perm + '.' + parent);
+            if(!$parentelem.is(':checked')){
+               $parentelem.click();
```

Comment:
I tried that one first and it's not working. It's checked but not considered when saving. That's why I used the click event.